docs/CommonMethods/meta-analysis/neurosynth.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Neurosynth
```

Comment:
Neurosynth-Compose

Reply:
Or maybe Neurosynth 2.0 to encompass the whole ecosystem.

Reply:
just updated it to neurosynth 2.0 and added a link to the original neurosynth site at the bottom!